src/modules/_nomodule/gl-context.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function runInBrowserContext(input, callback, step, options) {
+
+    // to ignore this from getting browserified
+    const puppeteer = eval('require')('puppeteer');
+
+    //Stripped down version of options which is serializable
+    var minOptions = require("lodash").cloneDeep(options);
+    minOptions.step = undefined;
+
+    var obj = { input: input, modOptions: minOptions }
+
+    puppeteer.launch().then(function(browser) {
+        browser.newPage().then(page => {
+            /* Maybe there is a better way to this, loading the page coz localstorage API
+            is not available otherwise */
+            page.goto("https://google.com").then(() => {
+                page.addScriptTag({ path: require('path').join(__dirname, '../../../dist/image-sequencer.js') }).then(() => {
+                    page.evaluate((options) => {
+                        return new Promise((resolve, reject) => {
+                            var sequencer = ImageSequencer();
+                            sequencer.loadImage(options.input.src);
+                            sequencer.addSteps('fisheye-gl', options.modOptions);
```

Comment:
Oh, is this potentially needing to be generalized?

Reply:
Oh yeah! Sorry about this, lol.